nx-X11/config/cf/host.def

```diff
@@ -133,18 +133,10 @@ XCOMM $XFree86: xc/config/cf/xf86site.def,v 3.186 2003/06/25 18:06:22 eich Exp $
 #define BuildLibraries			YES
 #endif
 
-#define BuildDamageLibrary              YES
-
-#define BuildXfixesLibrary              YES
-
-#define BuildCompositeLibrary           YES
-
 #define BuildRandRLibrary               YES
 
 #define BuildDocs                       NO
 
-#define BuildComposite			YES
-
```

Comment:
Note that  `nx-X11/config/cf/X11.tmpl` sets `BuildComposite` to `(BuildXfixes && BuildDamage)` if not defined. As none are defined here, it will default to `YES`.